specification/apimanagement/resource-manager/readme.az.md

```diff
@@ -0,0 +1,40 @@
+## AZ
+
+These settings apply only when `--az` is specified on the command line.
+
+``` yaml $(az) && $(target-mode) == 'core'
+tag: package-preview-2021-01
+az:
+    extensions: apim
+    namespace: azure.mgmt.apimanagement
+    package-name: azure-mgmt-apimanagement
+az-output-folder: $(azure-cli-folder)/src/azure-cli/azure/cli/command_modules/apim
+python-sdk-output-folder: $(az-output-folder)/vendored_sdks/apimanagement
+cli:
+    cli-directive:
+        - where:
+            group: "*"
+            op: "*"
+          hidden: true
```

Comment:
why hide all

Reply:
This is just for onboarding to the code auto generation. It will change the folder structure and add files to CLI but does not add any actually command. Adding actual commanding using codegen will come later